src/main/java/bio/terra/cli/command/app/passthrough/ToolCommand.java

```diff
@@ -0,0 +1,78 @@
+package bio.terra.cli.command.app.passthrough;
+
+import bio.terra.cli.app.utils.LocalProcessLauncher;
+import bio.terra.cli.businessobject.Config.CommandRunnerOption;
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.command.shared.BaseCommand;
+import bio.terra.cli.command.shared.options.WorkspaceOverride;
+import bio.terra.cli.exception.SystemException;
+import bio.terra.cli.exception.UserActionableException;
+import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+
+public abstract class ToolCommand extends BaseCommand {
+  private static final Logger logger = LoggerFactory.getLogger(ToolCommand.class);
+  /** Return the name used to invoke this command in the shell. */
+  public abstract String getExecutableName();
+
+  /** A non-operational argument, such as version, to check command availability. */
+  public String getVersionArgument() {
+    return "version";
+  }
+
+  @CommandLine.Mixin protected WorkspaceOverride workspaceOption;
+
+  @CommandLine.Unmatched protected final List<String> command = new ArrayList<>();
+
+  /** Pass the command through to the CLI Docker image. */
+  @Override
+  protected void execute() {
+    if (CommandRunnerOption.LOCAL_PROCESS == Context.getConfig().getCommandRunnerOption()
+        && !toolIsInstalled()) {
+      throw new UserActionableException(
+          String.format(
+              "Unable to launch %s. Please verify it is installed and included in the PATH.",
```

Comment:
I think it would be nice to print a url, so the message is more actionable. This could be helpful if user isn't technical.

At first I was thinking you could link to the README installation section. Then I was thinking it could be nice to link to installation instructions directly. Could `ToolCommand` have a method that each tools sets like:

```
String getInstallationUrl()
```

Also, I wonder if we should mention docker, in case the user might prefer docker over installing stuff manually.

```
Unable to launch %s. To install %s, see %s. Alternatively, run `terra config set app-launch DOCKER` to run %s inside a docker container.
```

Reply:
It's hard to explain the docker option in an error message, but adding the URLs is easy enough.